### PR TITLE
Secret Management via Sops

### PR DESCRIPTION
**Pull request by gshipunov**

### CODE REVIEW

**Comment by gshipunov** [comment]
@revol-xut I just have successfully deployed traffic-stop-box-3 with this, plz have a look, add the postgres magic and merge :).

**Comment by tanneberger** [comment]
Looks pretty gud to me. @oxapentane what postgres magic ?

File: modules/data-hoarder/clicky-bunty.nix [hunk 1]
```diff
@@ -24,8 +24,8 @@ in
 
         environment = {
           "POSTGRES" = "localhost:5432";
-          "POSTGRES_PASSWORD" = "/root/postgres_password"; #TODO: do it proper with sops
-          "SALT_PATH" = "/root/password_hash_salt"; #TODO: do it proper with sops
+          "SALT_PATH" = config.sops.secrets.postgres_password_hash_salt.path;
```
Comment (gshipunov) [inline modules/data-hoarder/clicky-bunty.nix:27]: @revol-xut for example check here that everything is correct, didn't test that

File: modules/data-hoarder/secrets.nix [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{ config, pkgs, ... }:
+{
+  sops.defaultSopsFile = ../../secrets/data-hoarder/secrets.yaml;
+  sops.age.sshKeyPaths = [ "/etc/ssh/ssh_host_ed25519_key" ];
+
+  sops.secrets = {
+    wg-seckey-staging = { };
+    wg-seckey = { };
+    postgres_password_hash_salt = {
```
Comment (gshipunov) [inline modules/data-hoarder/secrets.nix:9]: Also here and below

**Review by gshipunov — COMMENTED** [review]
@revol-xut here's the parts I'm talking about, also, if there's anything else we want to move to sops?

**Pull request merged**